src/Workspaces/Core/Portable/CodeGeneration/Symbols/CodeGenerationAbstractMethodSymbol.cs

```diff
@@ -32,7 +32,7 @@ public virtual ImmutableArray<AttributeData> GetReturnTypeAttributes()
         public abstract int Arity { get; }
         public abstract bool ReturnsVoid { get; }
         public abstract bool ReturnsByRef { get; }
-        public abstract bool ReturnsByReadonlyRef { get; }
+        public abstract bool ReturnsByRefReadonly { get; }
```

Comment:
Yes, my change changes this API. ReturnsByRef would return `true` regardless if `readonly` or not, and then there will be another method to check for `readonly`

Usage shows it is more convenient. But that yet may change. 
That is why we are not asking yet for a formal review of this API.


---
In reply to: [102998261](https://github.com/dotnet/roslyn/pull/17337#discussion_r102998261) [](ancestors = 102998261)